src/main.ts

```diff
@@ -65,10 +69,15 @@ class LocalTableland {
     this.#_cleanup();
 
     // Run a local hardhat node
-    this.registry = spawn("npx", ["hardhat", "node"], {
-      detached: true,
-      cwd: this.registryDir,
-    });
+    this.registry = spawn(
+      isWindows() ? "npx.cmd" : "npx",
+      ["hardhat", "node"],
+      {
+        // we can't run in windows if we use detached mode
+        detached: !isWindows(),
```

Comment:
It might be worth spending a little more time seeing if we can get away with running with `detached: false` in other systems, but I was having trouble with graceful shutdowns in pre 0.0.0 versions